app/vodic/vodic-responsive.css

```
@@ -0,0 +1,254 @@
+        justify-content: flex-end;
+        margin: auto;
+    padding-right: 13px; }
+ .own_website_sec_faqs_footer p{ color: #3C3E41; font-size: 14px; font-weight: 400; line-height: 24px; }
+
+ .mob_secton_four{ background-color: #ecf0f3 !important; }
+
+ .mob_secton_four .web_faq_data{ display: none !important; }
+.mob_secton_four .mob_faq_data{ display: block !important; }
+
+.mob_secton_four .fs_14px{ font-size: 14px !important; } 
+.mob_secton_four .detail_list p, .mob_secton_four .detail_list ul li{ font-size: 16px; font-size: 400; color: #3C3E41; line-height: 24px; }
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)